dynsampler.go

```diff
@@ -13,4 +13,12 @@ type Sampler interface {
 	// should call it with whatever key you choose to use to partition traffic
 	// into different sample rates.
 	GetSampleRate(string) int
+
```

Comment:
for consistency, these newlines should go away or a newline should be inserted between `Start` and `GetSampleRate`. (I think I prefer the latter but there's already a mix in this package.)